docs/pages/reference/cli.mdx

```diff
@@ -6,8 +6,8 @@ h1: Command Line (CLI) Reference
 
 Teleport is made up of three CLI tools.
 
-- [teleport](#teleport): The daemon that runs the Auth Service, Proxy Service, and other Teleport services.
-- [tsh](#tsh): A tool that lets end users interact with Teleport.
+- [teleport](#teleport): A CLI tool that supports the Teleport access plane.
```

Comment:
It's a little odd to hear `teleport` described as a CLI tool, but I suppose it's not entirely wrong to say so. `daemon` was definitely more misleading as Teleport runs as a foreground process. So perhaps this will do for now.

Reply:
I struggled with the wording on this as well, but felt that it was best to lean into CLI ability given the context of the doc. Maybe it'd be better to say "A CLI tool and foreground process..." just to be more accurate?